### DISCUSSION TITLE
How to generate barer token for admin user

### DISCUSSION THREAD

**Discussion by lynas** [zitadel/zitadel#6227] [Q&A] [answered]
Where/How do I generate the barer token for the admin endpoint?

```
curl -L -X POST 'https://$ZITADEL_DOMAIN/admin/v1/import' \
-H 'Content-Type: application/json' \
-H 'Accept: application/json' \
-H 'Authorization: Bearer <TOKEN>' \
--data-raw '{
```

**Comment by lynas** [comment, marked as answer]
You have multiple options:

1) you use the token your user has in console
2) you can use tools like insomnia/postman which can do a proper oauth code flow
3) you create a service account with a static token (do not forget to give management access for zitadel to that account) https://zitadel.com/docs/guides/integrate/pat

**Reply by fforootd** [reply]
For reference this was discussed here https://discord.com/channels/927474939156643850/927866013545025566/1130524193881063455
